packages/@aws-cdk/aws-events-targets/lib/ecs-task.ts

```diff
@@ -49,28 +49,50 @@ export interface EcsTaskProps {
    * (Only applicable in case the TaskDefinition is configured for AwsVpc networking)
    *
    * @default A new security group is created
+   * @deprecated use securityGroups instead
    */
   readonly securityGroup?: ec2.ISecurityGroup;
+
+  /**
+   * Existing security groups to use for the task's ENIs
+   *
+   * (Only applicable in case the TaskDefinition is configured for AwsVpc networking)
+   *
+   * @default A new security group is created
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
 }
 
 /**
  * Start a task on an ECS cluster
  */
 export class EcsTask implements events.IRuleTarget {
   public readonly securityGroup?: ec2.ISecurityGroup;
```

Comment:
Since this is public, not setting securityGroup would be considered a breaking change as there might be existing logic that uses this property.

Reply:
👍 thanks! that makes sense to me, fixed